### PR TITLE
Admin - Switch Import

### LINKED ISSUE #4091: GET /api/v1/switches/bulk_{action}
We need some bulk action for switches, the most important would be to change the switch group. See #1095.

### LINKED ISSUE #4825: web admin: import switches from CSV is missing
Import switches from a CSV file is not available in new web admin. It was the case in old admin.

**Expected behavior**
Possibility to import switches from a CSV.

### PR DESCRIPTION
**Pull request by satkunas**
# Description
Implement CSV import for switches.

# Impacts
n/a

# Issue
fixes #4825
fixes #4091

# Delete branch after merge
YES

# Checklist
(REQUIRED) - [yes, no or n/a]
- [n/a] Document the feature
- [n/a] Add unit tests
- [n/a] Add acceptance tests (TestLink)

# NEWS file entries
(REQUIRED, but may be optional...)
## New Features
* Switch CSV Import.
* Export CSV and search column control throughout entire Configuration.

## Bug Fixes
* web admin: import switches from CSV is missing (#4825)
* GET /api/v1/switches/bulk_{action} (#4091)

### CODE REVIEW

**Comment by satkunas** [comment]
@jrouzierinverse switch/bulkImport endpoint needed resembling the one in Nodes and Users.

There's currently a shim in place here: https://github.com/inverse-inc/packetfence/pull/5034/files#diff-fb264530f0ccc4232739a032f2e08cc7R99

**Comment by julsemaan** [comment]
Ensure that we can import id, description and group

Also needs to be rebased and then @jrouzierinverse can review + merge

**Comment by satkunas** [comment]
@julsemaan @jrouzierinverse rebased. Ready for review.

**Comment by nqb** [comment]
@julsemaan  and @jrouzierinverse, could you review this PR ? Thanks.

**Comment by jrouzierinverse** [comment]
This should be reworked to use the bulk_import interface when #4212 is merged.

File: html/pfappserver/root/static.alt/src/views/Configuration/_store/switches.js
Comment (cgx) [inline html/pfappserver/root/static.alt/src/views/Configuration/_store/switches.js]: This is redundant. `return bulkImport(_items)` is enough.

**Comment by nqb** [comment]
Would be nice to have in 10.1, changing milestone.

**Comment by satkunas** [comment]
> This should be reworked to use the bulk_import interface when #4212 is merged.

I will update once it's merged.

**Comment by satkunas** [comment]
@jrouzierinverse @cgx updated, ready to be re-reviewed.

File: html/pfappserver/root/static.alt/src/views/Configuration/_components/SwitchesImport.vue
Comment (cgx) [inline html/pfappserver/root/static.alt/src/views/Configuration/_components/SwitchesImport.vue]: `dryRun` is not used.

File: html/pfappserver/root/static.alt/src/views/Configuration/_components/SwitchesImport.vue
Comment (cgx) [inline html/pfappserver/root/static.alt/src/views/Configuration/_components/SwitchesImport.vue]: You don't have to wrap your call inside another promise. Just do

```
importPromise (payload) {
  return this.$store.dispatch('$_switches/bulkImport', payload)
}
```

File: html/pfappserver/root/static.alt/src/views/Configuration/_components/SwitchesImport.vue
Comment (cgx) [inline html/pfappserver/root/static.alt/src/views/Configuration/_components/SwitchesImport.vue:62]: No need to specify the name since it's already defined in the component itself.

File: html/pfappserver/root/static.alt/src/views/Configuration/_components/SwitchesImport.vue
Comment (cgx) [inline html/pfappserver/root/static.alt/src/views/Configuration/_components/SwitchesImport.vue]: This will affect all `b-nav` instances, so make sure it's required and move it to `src/styles/_nav.scss`.

**Review by cgx — CHANGES_REQUESTED** [review]
@satkunas Some minor changes to do on the frontend.

**Comment by satkunas** [comment]
@cgx changes pushed

**Review by cgx — APPROVED** [review]

**Pull request merged**
